### PR TITLE
Add multi-module sample project

### PR DESCRIPTION
**Pull request by takecare**
As setting up SAP on a multi-module project can be a bit tricky (or not very straight-forward) a sample project showcasing this sort of setup can be very helpful for people that want to use the plugin under such conditions.

The multi-module sample is a copy of the already existing sample, with an added (Java) module and the required changes to the SAP setup.

### CODE REVIEW

File: sample-multi-module/build.gradle [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            jcenter()
+        }
+    }
+
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
```
Comment (rock3r) [inline sample-multi-module/build.gradle]: Nit: extra whitelines

File: sample-multi-module/build.gradle [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    ext.kotlin_version = '1.2.30'
+
+    repositories {
+        google()
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath "gradle.plugin.io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.0.0.RC6-3"
```
Comment (rock3r) [inline sample-multi-module/build.gradle]: RC6-4 is out FYI

File: sample-multi-module/team-props/tasks.gradle [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+task check(dependsOn: subprojects.collect { p ->
+    "$p.name:check"
+})
+
+task assemble(dependsOn: subprojects.collect { p ->
+    "$p.name:assemble"
+})
+
+task build(dependsOn: ['assemble', 'check'])
```
Comment (rock3r) [inline sample-multi-module/team-props/tasks.gradle]: Are we sure we need these three tasks? Isn't it the default behaviour?

Comment (takecare) [reply sample-multi-module/team-props/tasks.gradle]: right. the way we refer to the tasks just needs some changing.

File: sample-multi-module/build.gradle (hunk 2 above)
Comment (takecare) [reply sample-multi-module/build.gradle]: seems to be breaking the plugin 🙀

Comment (takecare) [reply sample-multi-module/build.gradle]: tracked in #102

**Pull request merged**